sixtyfps_compiler/generator/rust.rs

```diff
@@ -883,6 +883,7 @@ fn compile_expression(e: &Expression, component: &Rc<Component>) -> TokenStream
                 quote!(#window_ref.scale_factor)
             }
             BuiltinFunction::Debug => quote!((|x| println!("{:?}", x))),
+            BuiltinFunction::Format => quote!(|x| format!("{:?}", x)),
```

Comment:
Remember this is  supposed to return a function with two arguments.  If we want to support variadic, this cannot be done here and need to be done in  the handler for `CallFunction` like `BuiltinFunction::SetFocusItem` this is not ideal and we probably should refactor that.